### PR TITLE
Redesign mod customization menu

### PR DESCRIPTION
**Pull request by Reco1I**
In this PR the mod customization menu (in code `InGameSettingsMenu`) is redesigned following the design line from #352.

- [x] Depends on #352

### CODE REVIEW

**Comment by Reco1I** [comment]
To unify components theme it is necessary to change App theme, that is some old fragments (such as beatmap properties fragment) are outdated due to visual inconsistencies. May we need to change them eventually.

Marking this PR as done for now.

File: src/com/edlplan/ui/fragment/InGameSettingMenu.kt [hunk 1]
```diff
@@ -213,34 +190,19 @@ class InGameSettingMenu : BaseFragment() {
                     progress: Int,
                     fromUser: Boolean
                 ) {
-                    findViewById<TextView>(R.id.brightPreviewText)!!.text = progress.toString()
                     findViewById<TextView>(R.id.bgBrightnessText)!!.text = "$progress%"
-                    val clamped = FMath.clamp(255 * (progress / 100f), 0f, 255f).roundToInt()
-                    findViewById<View>(R.id.brightnessPreview)!!.setBackgroundColor(
-                        0xFF shl 24 or (clamped shl 16) or (clamped shl 8) or clamped
-                    )
                 }
```
Comment (Rian8337) [inline src/com/edlplan/ui/fragment/InGameSettingMenu.kt:194]: Is there a reason for removing the brightness preview? A "redesign" should not remove features.

Comment (Reco1I) [reply src/com/edlplan/ui/fragment/InGameSettingMenu.kt:194]: That was not a preview.

File: res/layout/mod_customization_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+                        android:textSize="12sp" />
+
+                    <SeekBar
+                        android:id="@+id/changeSpeedBar"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_columnSpan="2"
+                        android:paddingVertical="12dp"
+                        android:saveEnabled="false"
+                        tools:progress="50" />
```
Comment (Rian8337) [inline res/layout/mod_customization_fragment.xml:161]: Missing `max` property (required so that speed multiplier caps at 2.00x)

**Review by Rian8337 — CHANGES_REQUESTED** [review]

File: src/com/edlplan/ui/fragment/InGameSettingMenu.kt [hunk 3]
```diff
@@ -550,14 +506,14 @@ class InGameSettingMenu : BaseFragment() {
         val layout = findViewById<View>(R.id.fullLayout)
         layout?.animate()?.cancel()
         layout?.animate()
-            ?.translationY(findViewById<LinearLayout>(R.id.optionBody)!!.height.toFloat())
+            ?.translationX(-(350f.dp.toFloat()))
```
[the remark this answers is not in the document]
Comment (Reco1I) [reply src/com/edlplan/ui/fragment/InGameSettingMenu.kt:509]: No, this matches the panel width which is 350 in DP metric

**Pull request merged**